app/models/user.rb

```diff
@@ -1,3 +1,14 @@
 class User < ApplicationRecord
   has_many :reviews
+
+  def highlighted_reviews
+    ordered = self.reviews.order(rating: :desc)
```

Comment:
Would it be possible to use a limit statement to minimize the work the database has to do, and the information we recieve?

Reply:
Could be wrong here, but I'm not certain that limit would reduce the db work required (i think the work is done ordering the set).  I wonder whether using 'minimum' and 'maximum' rather than 'order' might be less work for the db...

Reply:
Actually, the min/max is arguably as much work as order and it's more complicated.  Min requires finding the min value, then going back through the set to find the name with the min value, then return the first value from that list of  min values (if there are multiple with same min).  Max would require same.  I actually think limit would increase work load because you'd have to sort twice - once for best and again for worst.  We can discuss though if you have additional thoughts.